Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:2.7-slim
+LABEL "project.home"="https://github.com/BitBotFactory/poloniexlendingbot"
+
+#
+# Build: docker build -t <your_id>/pololendingbot .
+# Run: docker run -d -v /pololendingbot_data:/data -p 8000:8000 <your_id>/pololendingbot
+#
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r ./requirements.txt
+
+COPY . .
+
+VOLUME /data
+
+RUN ln -s /data/market_data market_data; \
+    ln -s /data/log/botlog.json www/botlog.json
```

Comment:
No, no configuration changes in config, web server or log json. All this are located in /data (could be a extra docker data volume or a local dir) and are safe from container changes (e.g remove and create a new one using new image version)
Directory structure in /data is:
/data
    |- marked_data
    |       | - marked data files
    |- log
    |       | - botlog.json
    |- config
    |        | - default.cfg